tensorboard/webapp/feature_flag/views/feature_flag_page.ng.html

```diff
@@ -16,16 +16,31 @@
 -->
 
 <div>
-  <table>
-    <ng-container *ngFor="let flag of getFlagKeys();">
+  <div class="message">
+    <h2 class="warning">WARNING: EXPERIMENTAL FEATURES AHEAD!</h2>
+    By enabling these features, you could put the application in an unusable
+    state or expose yourself to untested features or potential bugs.
+  </div>
+  <table class="feature-flag-table">
+    <ng-container *ngFor="let state of featureFlagStatuses;">
       <tr>
-        <mat-checkbox
-          [checked]="featureFlags[flag]"
-          (change)="flagChanged(flag)"
-          >Enabled</mat-checkbox
-        >
-        {{flag}}
+        <td>
+          <mat-select
+            [value]="state.status"
+            (selectionChange)="flagChanged.emit({flag: state.flag, status: $event.value})"
+          >
+            <mat-option value="default"
+              >Default - {{state.defaultValue}}</mat-option
```

Comment:
Is it possible to use same wording? 
Rather than:
"Default - true", "Default - false", "Enabled", "Disabled"
Have:
"Default - Enabled", "Default - Disabled", "Enabled", "Disabled".

Reply:
I'm not sure that a direct mapping scales well if we decide to support non boolean flags. I'm going to instead create a formatter that will convert `true`/`false` values to `Enabled`/`Disabled` while also allowing us to extend formatting in the future.